storage/collections/ibft.go

```diff
@@ -0,0 +1,34 @@
+package collections
+
+import (
+	"github.com/bloxapp/ssv/ibft/proto"
+	"github.com/bloxapp/ssv/storage"
+	"go.uber.org/zap"
+)
+
+// Ibft struct
+type Ibft struct {
```

Comment:
@nivBlox is this a public struct? if so maybe give it a different name.. if it's private than first letter shouldn't be capitalized .

Reply:
@alonmuroch will change to IbftStorage